bittensor/metagraph.py

```diff
@@ -135,6 +136,22 @@ def __init__(self):
         self.uid_for_pubkey = {}
         self.index_for_uid = {}
 
+    def write_to_file(self, filepath: str ):
+        json_data = {
+            'block': self.block,
+            'tau': self.tau.tolist(),
+            'n': self.n,
+            'uids': self.uids.tolist(),
+            'indices': self.indices.tolist(),
+            'stake': self.stake.tolist(),
+            'lastemit': self.lastemit.tolist(),
+            'W': self.W.tolist(),
+            'neurons': [ {'uid': n.uid, 'ip': n.address, 'port': n.port, 'ip_type': n.ip_type, 'modality': n.modality, 'hotkey': n.public_key} for n in self.neurons]
+        }
+        with open( filepath, 'w') as fp:
```

Comment:
Should we overwrite the file each time or augment?

Reply:
the file name is appended with the block height